core/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'bintray-release'
 
 dependencies {
-    compile 'com.android.support:support-v4:22.1.1'
+    compile 'com.android.support:design:22.2.0'
```

Comment:
hmm slippery slope @ouchadam means develop is not in a position where you can shipit != continuous integration